api/app/serializers/spree/v2/storefront/address_serializer.rb

```diff
@@ -0,0 +1,24 @@
+module Spree
+  module V2
+    module Storefront
+      class AddressSerializer < BaseSerializer
+        set_type :address
+
+        attributes :firstname :lastname, :address1, :address2, :city, :zipcode, :phone, :state_name,
+                   :company
+
+        attribute :state_code do |address|
+          address.state.abbr
```

Comment:
Can we use delegation here in order to prevent breaking LoD?
Same goes for country name & country iso3